client/init/init.lua

```diff
@@ -30,13 +30,16 @@ AddEventHandler('onClientResourceStart', function(resource)
 		voiceMode = mode - 1
 	})
 
+	local radioChannel = LocalPlayer.state.radioChannel
```

Comment:
You can just do this and it will not add as much unpretty logic

`local radioChannel = LocalPlayer.state.radioChannel or 0`

Reply:
Yeah true, I'll add that